packages/eslint-plugin-turbo/lib/utils/getEnvVarDependencies.ts

```diff
@@ -0,0 +1,44 @@
+import findTurboConfig from "./findTurboConfig";
+import type { TurboConfig } from "../types";
+
+function findDependsOnEnvVars({
+  dependencies,
+}: {
+  dependencies?: Array<string>;
```

Comment:
TS q... iiuc `?` means that the argument is optional. Could we get by with assigning a default value to the arg (and therefore prevent the `if` check in the function body? (not a blocker, just curious!

Reply:
This is a transitive-to-user-input `?`, so at _some_ point it'll have to be optional. We can choose where we put the if statement, basically.